scenes/game/local_game.gd

```diff
@@ -85,6 +85,7 @@ func create_event(event_type : Enums.EventType, event_player : Enums.PlayerId, n
 		"reason": reason,
 		"extra_info": extra_info,
 		"extra_info2": extra_info2,
+		"extra_info3": extra_info3,
```

Comment:
I really should have made this more variable, lol...
This is fine though, just leave it.
Surely we won't need any more than 3 extra things, right?